site/docs/components/slider.mdx

```diff
@@ -51,6 +58,9 @@ Steps are selectable values along a slider range and can be visually indicated b
 
 Where the range of values is not obvious, sliders should be labeled with minimum and maximum values. An optional center label can also be used to denote a middle value or as a description.
 
+### Progress track
+
+A filled progress track is used in the progress slider. It follows the path of the regular track to indicate where the handle has been dragged from.
```

Comment:
Is "where the handle has been dragged from" clear enough – could it be interpreted as "from it's last position" rather than "from the lowest/left value in the range"?

Reply:
Good suggestion; I've updated the wording - hope it's a little more clear.